modules/ols-about-rhelai.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+// about/ols-about-openshift-lightspeed.adoc
```

Comment:
```suggestion
// Module included in the following assemblies:

// * about/ols-about-openshift-lightspeed.adoc
```

This is an ultimately not-very-important nit, but I did notice that the other modules in this PR (technically outside of these changes/scope) are either missing this "Module included..." metadata or are missing its `*` before the file path and blank line (per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)). If you like, suggest syncing all of those up.

Reply:
done